sympy/integrals/tests/test_integrals.py

```diff
@@ -593,7 +593,7 @@ def test_integrate_max_min():
     assert integrate(Min(exp(x), exp(-x))**2, x) == Piecewise( \
         (exp(2*x)/2, x <= 0), (1 - exp(-2*x)/2, True))
     # issue 7907
-    c = symbols('c', real=True)
+    c = symbols('c', extended_real=True)
     int1 = integrate(Max(c, x)*exp(-x**2), (x, -oo, oo))
     int2 = integrate(c*exp(-x**2), (x, -oo, c))
     int3 = integrate(x*exp(-x**2), (x, c, oo))
```

Comment:
This test gives different answers depending on whether c is declared as real or extended_real. Here I've just changed it to extended_real to match previous behaviour but maybe something needs fixing in integrals. The difference for int3 is:
```julia
In [16]: c = symbols('c', real=True)                                                                                              

In [17]: integrate(x*exp(-x**2), (x, c, oo))                                                                                      
Out[17]: 
  ⎛                     2⎞               
  ⎜                   -c ⎟               
  ⎜                2⋅ℯ   ⎟               
c⋅⎜-2⋅√π⋅erfc(c) + ──────⎟               
  ⎝                  c   ⎠   √π⋅c⋅erfc(c)
────────────────────────── + ────────────
            4                     2      

In [18]: c = symbols('c', extended_real=True)                                                                                     

In [19]: integrate(x*exp(-x**2), (x, c, oo))                                                                                      
Out[19]: 
   2
 -c 
ℯ   
────
 2  
```
For int2 the difference is:
```julia
In [20]: c = symbols('c', real=True)                                                                                              

In [21]: integrate(c*exp(-x**2), (x, -oo, c))                                                                                     
Out[21]: 
√π⋅c⋅(2 - erfc(c))
──────────────────
        2         

In [22]: c = symbols('c', extended_real=True)                                                                                     

In [23]: integrate(c*exp(-x**2), (x, -oo, c))                                                                                     
Out[23]: 
√π⋅c⋅erf(c)   √π⋅c
─────────── + ────
     2         2  
```

Reply:
hmm the negative sign is missing.

Reply:
I think the result above for int2 with extended_real=True is correct. It is also the same as the result on master with real=True. The result for real=True on this PR is incorrect:
```julia
In [1]: c = symbols('c', real=True)                                                                                               

In [2]: integrate(c*exp(-x**2), (x, -oo, c))                                                                                      
Out[2]: 
√π⋅c⋅(2 - erfc(c))
──────────────────
        2 
```
That's a separate issue from this PR though because I get the same on master if I declare c to be finite:
```julia
In [1]: c = symbols('c', real=True, finite=True)                                                                                  

In [2]: integrate(c*exp(-x**2), (x, -oo, c))                                                                                      
Out[2]: 
√π⋅c⋅(2 - erfc(c))
──────────────────
        2
```